README.md

```diff
@@ -105,6 +105,8 @@ Version: 2.1.0
   BtoA.dump for the data received from B
  in both cases a raw hexdump is saved which can be recovered later
  running 'xxd -p -c 16 -r <raw hexdump file>'. See man xxd(1)
+~
+ -f <prefix> adds a prefix to both the dump file names
```

Comment:
See the comments about the help command)

Reply:
Done